### DISCUSSION TITLE
Fuzzyfinder initial error

### DISCUSSION THREAD

**Discussion by themuuln** [LazyVim/LazyVim#3050] [General]
Everytime I start my neovim and try to search something via using fzf, this error appears. The requirements are already downloaded, and even the fzf is working fine. How can i fix this initial message?
here's my config: https://github.com/themuuln/nvim-lazy

![image](https://github.com/LazyVim/LazyVim/assets/75017829/78ca7a25-d35d-4e52-a804-94d4333d5e31)

**Comment by dpetka2001** [comment]
That error seems to be because of the plugin not being able to build correctly. Are you sure you have the [Windows requirements](https://github.com/nvim-telescope/telescope-fzf-native.nvim?tab=readme-ov-file#cmake-windows-linux-macos) mentioned in the plugin's README? I tried your dotfiles on Linux and don't get any error, so it doesn't seem to be a configuration issue.

**Reply by themuuln** [reply]
yeah i've installed the requirements. Do you use Codeium btw?

**Reply by dpetka2001** [reply]
I used to, but now I'm testing Neocodeium.

**Reply by themuuln** [reply]
I got a few problems to use codeium, currently im using both vscode, neovim just because can't setup codeium correctly in my neovim. I still don't know what's the difference between codeium.nvim, codeium.vim, even though i already saved my api key on codeium.nvim, this error messages still appear while im coding, can you help me with that?
![Screenshot 2024-04-24 164402](https://github.com/LazyVim/LazyVim/assets/75017829/46b1680b-6501-4b4b-bce6-9c11a37a56eb)
![Screenshot 2024-04-24 212540](https://github.com/LazyVim/LazyVim/assets/75017829/04efda48-fbea-4350-ae85-685c9f57d56e)

**Reply by themuuln** [reply]
or just my session is just expired? should i save my api everytime i start my neovim?

![image](https://github.com/LazyVim/LazyVim/assets/75017829/793814bb-9da4-4b74-b736-767bb747f69e)

**Reply by dpetka2001** [reply]
I don't use `codeium.nvim`. I used `codeium.vim` for a while and now `neocodeium` which is based on `codeium.vim`. `codeium.nvim` operates as a source for `nvim-cmp` and I didn't like that, so I used `codeium.vim`, which only shows virtual text. I've just saved my API key only once, the first time I did `Codeium Auth`. After that, I don't need to enter it again.

**Reply by dpetka2001** [reply]
I'm seeing relevant issues in `codeium.nvim` repo, so it might be some kind of regression in a recent update. Try to install some older version of the plugin and see if that works for you.

**Reply by themuuln** [reply]
i just swapped my codeium with `codeium.vim` but whenever i try to enter my api key the modal thing blocks the url, how can i have access to the url? i tried to enter `:Notifications` command to see my history, but after enter my api key that says wrong key: 
![image](https://github.com/LazyVim/LazyVim/assets/75017829/f7041be7-12e6-4c84-98b3-d6097f7e0b43)

**Reply by themuuln** [reply]
![image](https://github.com/LazyVim/LazyVim/assets/75017829/0a32111a-f0fe-4731-8a43-2bd27673ef24)
Even though i enter the correct api, this error occurs, so the Codeium has problem?

**Reply by dpetka2001** [reply]
Not sure. From what I've read in the issues, it seems to be happening in files that have lots of lines and the request to the server ends up with lots of body data. You better ask in the codeium repo instead. I'm not familiar with their codebase. 

About `fzf-native` I installed `cmake` via `scoop` and then the plugin built correctly on Windows. Might be something you want to try out.

**Reply by themuuln** [reply]
Thank you
